### PR TITLE
FM2-361: Get rid of warnings by adding @Deprecated annotation

### PR DESCRIPTION
**Pull request by podorvanova**
## Description of what I changed

This PR add `@Deprecated` annotation to isRetired methods which eliminates 3 warnings during project building.

## Issue I worked on

see https://issues.openmrs.org/browse/FM2-361

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by Medhavi-16** [comment]
@podorvanova Please add the ticket ID in your PR name. https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: api/src/main/java/org/openmrs/module/fhir2/model/FhirDurationUnitMap.java [hunk 1]
```diff
@@ -84,6 +84,7 @@
 	private String uuid = UUID.randomUUID().toString();
 	
 	@Override
+	@Deprecated
```
Comment (Medhavi-16) [inline api/src/main/java/org/openmrs/module/fhir2/model/FhirDurationUnitMap.java:87]: Why should we generate a warning?

Comment (podorvanova) [reply api/src/main/java/org/openmrs/module/fhir2/model/FhirDurationUnitMap.java:87]: isRetired method is deprecated in core OpenMRS now. That creates warnings during project building. We can not just remove the method, so we need to mark it with `@Deprecated` annotation.

Comment (Medhavi-16) [reply api/src/main/java/org/openmrs/module/fhir2/model/FhirDurationUnitMap.java:87]: Thanks, Some of the models is used for mapping purpose and used in this module only. @ibacher @varung-31 your thought?

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
@podorvanova Thanks for this.

**Pull request merged**